guides/Postgres Query Optimization + Diagnostic Tooling.md

```diff
@@ -0,0 +1,68 @@
+# Postgres Query Optimization + Diagnostic Tooling
+
+
+
+PostgreSQL is a database management system that utilizes SQL and includes features like foreign keys, updatable views, and various query optimization tools. This article will outline some of the query optimization techniques in Postgres using diagnostic tools.
+<br />
+## Scope:
+Query optimization involves identifying the most efficient way to execute a query, considering the structure of the database and available indexes. The Postgres query optimizer can perform this task. Proper optimization techniques can help reduce costs and enhance environmental sustainability by using resources more efficiently.
+Postgres Query optimiser:
```

Comment:
Could this be removed and have the next sentence start with just "The Postgres Query Optimizer"? Also should this all be capitalized? Can we have a backlink here?